mix.exs

```diff
@@ -41,11 +41,13 @@ defmodule Membrane.RTMP.Mixfile do
       {:membrane_core, "~> 0.10.0"},
       {:unifex, "~> 1.0"},
       {:membrane_h264_ffmpeg_plugin, "~> 0.21.1"},
-      {:membrane_aac_plugin, "~> 0.12.1"},
+      {:membrane_aac_plugin, github: "membraneframework/membrane_aac_plugin", override: true},
```

Comment:
Note that we have switched to the aac plugin, so the timestamps of our sample .msr file are not being overridden by the aac parser (see last commit). We are still waiting for the next release on hex.

Reply:
Ok, I released the latest changes on hex.